src/templates/base.html

```diff
@@ -47,19 +54,16 @@
         </div>
     </nav>
 
-    <div class="jumbotron">
-        <div class="container">
-            <h1>Uh-oh!</h1>
-            <h2>404 Error - Page Not Found</h2>
-            <br>
-            <p><a class="btn btn-primary btn-lg" href="/" role="button">Home page &raquo;</a></p>
-        </div>
-    </div>
+    {% block body %}
+    {% endblock %}
 
     <!-- Bootstrap core JavaScript & jQuery -->
     <!-- Placed at the end of the document so the pages load faster -->
     <script src="https://code.jquery.com/jquery-3.1.1.js" integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA=" crossorigin="anonymous"></script>
     <script> window.jQuery || alert("jQuery could not be loaded."); </script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
Do you wanna use the non-minified version for testing/debug purposes then move to minified files for production?